biodatasets/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Define all the useful directories for the project.
+
+- ROOT_DIRECTORY: main directory
+
+- CURRENT_DIRECTORY: directory with all the code
+
+- CACHE_DIRECTORY: directory used to store the cache datasets
+"""
+from pathlib import Path
+
+CURRENT_DIRECTORY = Path(__file__).resolve().parent
+ROOT_DIRECTORY = CURRENT_DIRECTORY.parent
+
+CACHE_DIRECTORY = ROOT_DIRECTORY / "cache"
```

Comment:
The cache directory is often hidden so we can add a point:
```suggestion
CACHE_DIRECTORY = ROOT_DIRECTORY / ".cache"
```